typescript-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@moonbeam-network/api-augment",
-  "version": "0.3200.0",
+  "version": "0.3300.0",
```

Comment:
FTR: In general we are updating the TS API during release. This PR includes lots of changes to it that and not exclusively related to the additional account for inflation distribution. These changes are not required for tests to execute because the current setup is using the published version and not the local package.

Later, once the overhaul is in place, it should be easier and each PR should contain the updates to the TS API.

I'm not against merging the updates with this PR, the main consequence is it shows a bunch of files to review and most might not be relevant.